docs/source/onedal/algorithms/clustering/dbscan.rst

```diff
@@ -0,0 +1,67 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. re-use for math equations:
+.. |x| replace:: :math:`x`
+.. |y| replace:: :math:`y`
+
+.. _alg_dbscan:
+
+======
+DBSCAN
+======
+
+.. include:: ../../../includes/clustering/dbscan-introduction.rst
+
+------------------------
+Mathematical formulation
+------------------------
+
+.. _dbscan_c_math:
+
+Computation
+-----------
+Given the set :math:`X = \{x_1 = (x_{11}, \ldots, x_{1p}), \ldots, x_n = (x_{n1}, \ldots, x_{np})\}`
+of :math:`n` :math:`p`-dimensional feature vectors (further referred as observations),
+a positive floating-point number ``epsilon`` and a positive integer ``minObservations``,
+the problem is to get clustering assignments for each input observation, based on the definitions below [Ester96]_:
+two observations |x| and |y| are considered to be in the same cluster if there is a :term:`core observation` :math:`z`,
```

Comment:
I believe that the use of 

```
:term:`core observation`
```

will lead readers to the DBSCAN chapter in old (DAAL) interfaces, are we ok with this? I'd think we wouldn't want this to happen